### PR TITLE
Add advanced config option for logWrap

### PR DESCRIPTION
**Pull request by micha3lbrown**
Add Advanced config option to change display of log formatting.
`logWrap` only impacts logs and does not enable config wrapping at this
time.

Advances #3

### CODE REVIEW

File: android/app/src/main/kotlin/net/defined/mobile_nebula/Sites.kt [hunk 1]
```diff
@@ -251,6 +251,7 @@ class IncomingSite(
     val cipher: String,
     val sortKey: Int?,
     var logVerbosity: String?,
+    val logWrap: Boolean?,
```
Comment (nbrownus) [inline android/app/src/main/kotlin/net/defined/mobile_nebula/Sites.kt:254]: We can undo this and it's friends now that it isn't a side based config

File: lib/models/Site.dart [hunk 2]
```diff
@@ -11,7 +11,8 @@ import 'StaticHosts.dart';
 var uuid = Uuid();
 
 class Site {
-  static const platform = MethodChannel('net.defined.mobileNebula/NebulaVpnService');
+  static const platform =
```
Comment (nbrownus) [inline lib/models/Site.dart:14]: Looks like your IDE may have taken a large hammer and adjusted much whitespace. I don't find a lot of value in it personally. Happy to discuss further if others have strong feelings.

I'll continue after we resolve these first two

File: android/app/src/main/kotlin/net/defined/mobile_nebula/Sites.kt (hunk 1 above)
Comment (micha3lbrown) [reply android/app/src/main/kotlin/net/defined/mobile_nebula/Sites.kt]: Fixed! ✅

File: lib/models/Site.dart (hunk 2 above)
Comment (micha3lbrown) [reply lib/models/Site.dart]: I've pulled out most of the flutter formatting which should make salient changes easy to identify. 👍

File: lib/screens/SiteLogsScreen.dart [hunk 3]
```diff
@@ -25,6 +26,7 @@ class _SiteLogsScreenState extends State<SiteLogsScreen> {
   ScrollController controller = ScrollController();
   RefreshController refreshController = RefreshController(initialRefresh: false);
 
+  var settings = Settings();
```
Comment (nbrownus) [inline lib/screens/SiteLogsScreen.dart:29]: This should be a member of `_SiteLogsScreenState` to avoid any global state

Comment (nbrownus) [reply lib/screens/SiteLogsScreen.dart:29]: And here is proof that I do not know how to read

**Review by nbrownus — APPROVED** [review]

**Pull request merged**